pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/periodictask/ControllerPeriodicTask.java

```diff
@@ -116,6 +116,8 @@ protected void processTables(List<String> tableNamesWithType, Properties periodi
         processTable(tableNameWithType, context);
       } catch (Exception e) {
         LOGGER.error("Caught exception while processing table: {} in task: {}", tableNameWithType, _taskName, e);
+        _controllerMetrics.addMeteredTableValue(tableNameWithType + "." + _taskName,
+            ControllerMeter.PERIODIC_TASK_ERROR, 1L);
```

Comment:
assuming this will be resolved as `<table_name>.<OFFLINE|REALTIME>.<TaskName>`? (e.g. tableNameWithType here is dot-separated)

Reply:
actually it'll be resolved as `<rawTableName>_<OFFLINE|REALTIME>.<periodicTaskName>`. Have set the expression in the yml accordingly 
```
name=\"pinot.controller.(\\w+)_(\\w+).(\\w+).periodicTaskError\
```

Reply:
ok looks like `(\\w+)_(\\w+)` does the resolution to right most (e.g. if there's underscore in table name it will make the second `(\\w+)` the smallest matching group.